virttest/utils_misc.py

```diff
@@ -477,8 +478,7 @@ def log_line(filename, line):
                 pass
             _open_log_files[base_file] = open(log_file, "w")
         timestr = time.strftime("%Y-%m-%d %H:%M:%S")
-        line = unicode("%s: %s" % (timestr, line), errors="ignore")
-        line = line.encode(ENCODING, errors='ignore')
+        line = string_safe_encode(line)
```

Comment:
ENCODING = os.environ.get("PYTHONENCODING", "utf-8") this line(52) can be removed, 

line = string_safe_encode(line) --> line = string_safe_encode("%s: %s" % (timestr,line))